internal/cli/cluster/connectinfo.go

```diff
@@ -0,0 +1,494 @@
+	MysqlClientID                   string = "mysqlclient"
+	MysqlClientDisplayName          string = "mysqlclient (Python)"
+	MysqlClientInputName            string = "python_mysqlclient"
+	PyMysqlID                       string = "pymysql"
+	PyMysqlDisplayName              string = "PyMySQL"
+	PyMysqlInputName                string = "pymysql"
+	MysqlConnectorPythonID          string = "mysql_connector_python"
+	MysqlConnectorPythonDisplayName string = "MySQL Connector/Python"
+	MysqlConnectorPythonInputName   string = "mysql_connector_python"
+	MysqlConnectorJavaID            string = "mysql_connector_java"
+	MysqlConnectorJavaDisplayName   string = "MySQL Connector/Java"
+	MysqlConnectorJavaInputName     string = "mysql_connector_java"
+	GoMysqlDriverID                 string = "go_mysql_driver"
+	GoMysqlDriverDisplayName        string = "Go MySQL Driver"
+	GoMysqlDriverInputName          string = "go_mysql_driver"
+	NodeMysql2ID                    string = "node_mysql_2"
+	NodeMysql2DisplayName           string = "Node MySQL 2"
+	NodeMysql2InputName             string = "node.js_mysql2"
+	Mysql2RubyID                    string = "mysql2_ruby"
+	Mysql2RubyDisplayName           string = "Mysql2 (Ruby)"
+	Mysql2RubyInputName             string = "ruby_mysql2"
+	MysqliID                        string = "mysqli"
+	MysqliDisplayName               string = "MySQLi (PHP)"
+	MysqliInputName                 string = "php_mysqli"
+	MysqlRustID                     string = "mysql_rust"
+	MysqlRustDisplayName            string = "mysql (Rust)"
+	MysqlRustInputName              string = "rust_mysql"
+	MybatisID                       string = "mybatis"
+	MybatisDisplayName              string = "MyBatis"
+	MybatisInputName                string = "mybatis"
+	HibernateID                     string = "hibernate"
+	HibernateDisplayName            string = "Hibernate"
+	HibernateInputName              string = "hibernate"
+	SpringBootID                    string = "spring_boot"
+	SpringBootDisplayName           string = "Spring Boot"
+	SpringBootInputName             string = "spring_boot"
+	GormID                          string = "gorm"
+	GormDisplayName                 string = "GORM"
+	GormInputName                   string = "gorm"
+	PrismaID                        string = "prisma"
+	PrismaDisplayName               string = "Prisma"
+	PrismaInputName                 string = "prisma"
+	SequelizeID                     string = "sequelize"
+	SequelizeDisplayName            string = "Sequelize (mysql2)"
+	SequelizeInputName              string = "sequelize"
+	DjangoID                        string = "django"
+	DjangoDisplayName               string = "Django (django_tidb)"
+	DjangoInputName                 string = "django"
+	SQLAlchemyID                    string = "sqlalchemy"
+	SqlAlchemyDisplayName           string = "SQLAlchemy (mysqlclient)"
+	SqlAlchemyInputName             string = "sqlalchemy"
```

Comment:
For "django"/"sqlalchemy"/"sequelize", it implies the under driver. Then if we are going to have "sqlalchemy (pymysql)", what's the input name?

Reply:
I have changed `ID` and `InputName` and made them with `_driver` suffixes.